### DISCUSSION TITLE
Windows Package manager 1.2.10271

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1895] [Releases]
This release represents our Windows Package Manager 1.2 release candidate build for Windows 10 (1809+), and Windows 11.

Experimental features have been disabled in this release. We will follow this release with another Pre-release "developer" build at GitHub so users can continue with experimental features available.

## What's Changed
* Support for x64 on ARM64 by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1441
* Add colons after agreement labels by @lechacon in https://github.com/microsoft/winget-cli/pull/1440
* Spelling by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1447
* Update spelling.yml by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1448
* Update winget trouble shooting steps  by @ashpatil-msft in https://github.com/microsoft/winget-cli/pull/1454
* Show source identifier with `source list` by @lechacon in https://github.com/microsoft/winget-cli/pull/1455
* Fix crash when resource.pri is not present by @msftrubengu in https://github.com/microsoft/winget-cli/pull/1443
* Disable local manifest by default by @yao-msft in https://github.com/microsoft/winget-cli/pull/1453
* Update localization files by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1462
* doc/Settings.md: Add missing "a" by @me-and in https://github.com/microsoft/winget-cli/pull/1467
* Add expected installer return codes by @lechacon in https://github.com/microsoft/winget-cli/pull/1421
* Remove cancel optimization for packageQuery caller by @sreadingMSFT in https://github.com/microsoft/winget-cli/pull/1472
* Update Docs by @vedantmgoyal2009 in https://github.com/microsoft/winget-cli/pull/1471
* Update Docs [suggestions] by @vedantmgoyal2009 in https://github.com/microsoft/winget-cli/pull/1479
* Minor tweaks to spelling to fix issues by @denelon in https://github.com/microsoft/winget-cli/pull/1480
* Fix Roadmap and Documentation by @vedantmgoyal2009 in https://github.com/microsoft/winget-cli/pull/1468
* Fix Installer return code max value in schema by bad merge by @yao-msft in https://github.com/microsoft/winget-cli/pull/1482
* Source resilience by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1470
* Make list show correlated source even when no upgrade is available by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1481
* Handle single unknown version from REST source by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1483
* Fix debug build failure by @yao-msft in https://github.com/microsoft/winget-cli/pull/1485
* Fix logging of large REST response by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1489
* Create machinery for disabling experimental features by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1490
* Remove motw before calling IAttachmentExecute::Save by @yao-msft in https://github.com/microsoft/winget-cli/pull/1491
* Fix correlation issue by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1492
* Add sample project for calling COM interface by @lechacon in https://github.com/microsoft/winget-cli/pull/1449
* Minor bug fixes by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1494
* Update troubleshooting doc with recent error and findings by @ashpatil-msft in https://github.com/microsoft/winget-cli/pull/1495
* Localize the product name per recommendation by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1498
* Update localization files by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1499
* Make rename retry more frequently for longer, then try making a hardlink by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1497
* Disable test hooks fixes by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1501
* Exit the locale choice loop if the default localization is good enough by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1500
* Ensure available packages match with a single installed package by @lechacon in https://github.com/microsoft/winget-cli/pull/1473
* Updating Readme by @denelon in https://github.com/microsoft/winget-cli/pull/1507
* Add option to check fields requiring verified publisher in manifest p… by @yao-msft in https://github.com/microsoft/winget-cli/pull/1487
* Use correct Subexecution id when working on multiple packages by @ashpatil-msft in https://github.com/microsoft/winget-cli/pull/1504
* Changes for sample COM caller project by @lechacon in https://github.com/microsoft/winget-cli/pull/1514
* Fixed bug where the package name wouldn't be written during upgrade. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1517
* Fix nuget release pipeline by @yao-msft in https://github.com/microsoft/winget-cli/pull/1522
* Fix test task overrides by @lechacon in https://github.com/microsoft/winget-cli/pull/1526
* Fix unit tests in pipeline by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1529
* Add missing resource string by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1532
* Change format to long for installer success codes by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/1533
* Settings streams exchange semantics by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1534
* Added vsconfig file to automatically install required workloads. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1551
* Color contamination fix by @hackean-msft in https://github.com/microsoft/winget-cli/pull/1527
* Added validation of count argument for search/list. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1564
* closes #1569 by @exxmen in https://github.com/microsoft/winget-cli/pull/1580
* Split COM install command into download and install stages by @lechacon in https://github.com/microsoft/winget-cli/pull/1528
* Check enablement state for all output write operations by @hackean-msft in https://github.com/microsoft/winget-cli/pull/1581
* Move Issue Templates to Forms by @Trenly in https://github.com/microsoft/winget-cli/pull/1596
* The source column is no longer shown in the table if a source was specified (list/upgrade). by @jedieaston in https://github.com/microsoft/winget-cli/pull/1598
* Add projected winrt wrapper for server com interface and add tests by @sreadingMSFT in https://github.com/microsoft/winget-cli/pull/1373
* Add link to new discussion for questions by @Trenly in https://github.com/microsoft/winget-cli/pull/1617
* Package type dependency graph specs by @fzanollo in https://github.com/microsoft/winget-cli/pull/1340
* Rearrange choices for ARP changes by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1630
* Introduce Managed Interop and merged manifest classes to WingetUtil nuget  by @jamespik in https://github.com/microsoft/winget-cli/pull/1621
* Update doc with Machine-wide install information by @ashpatil-msft in https://github.com/microsoft/winget-cli/pull/1642
* Added progress indication to InstallMultiplePackages. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1613
* Installed package tracking by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1614
* Package type Dependency graph by @fzanollo in https://github.com/microsoft/winget-cli/pull/1341
* Refactor Source Interface to be more object oriented by @yao-msft in https://github.com/microsoft/winget-cli/pull/1631
* CurrentLocalization is now populated for dependency manifests. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1655
* Fix yaml manifest schema and parsing for ExpectedReturnCodes by @yao-msft in https://github.com/microsoft/winget-cli/pull/1646
* Fix incorrect processing of PackageMatchFields in rest client by @yao-msft in https://github.com/microsoft/winget-cli/pull/1659
* Fix interface casting in client dll. by @sreadingMSFT in https://github.com/microsoft/winget-cli/pull/1673
* Prevent duplicate installations in winget upgrade all by @yao-msft in https://github.com/microsoft/winget-cli/pull/1652
* Use package tracking data to correlate in CompositeSource by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1671
* Add product code and PFN to enable existing searches by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1677
* Added an option to select architecture to install. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1666
* Add preview concept to allow build time determination by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1679
* Print message when install technology is different in upgrade scenarios by @msftrubengu in https://github.com/microsoft/winget-cli/pull/1649
* Prevent circular dependency in global telemetry logger creation by @yao-msft in https://github.com/microsoft/winget-cli/pull/1674
* Made all installer types lowercase during `show`. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1689
* Use SFINAE to stop paths from logging without using u8string by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1697
* Made the Installation Disclaimer for the Microsoft Store bold. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1698
* remove plain by @MagnetarRocket in https://github.com/microsoft/winget-cli/pull/1700
* Make VT enablement/disablement work properly by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1699
* Create shared DLL for winget.exe and COM server by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1687
* Implement parallel downloads for COM scenarios by @lechacon in https://github.com/microsoft/winget-cli/pull/1588
* Updated Valijson to version 0.6. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1721
* Change download file name to match the installer URL by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/1722
* Updated solution to build under Visual Studio 2022 by @jedieaston in https://github.com/microsoft/winget-cli/pull/1720
* Update component governance manifest with valijson v0.6 commit by @yao-msft in https://github.com/microsoft/winget-cli/pull/1732
* Update source agreement market message to be more clear by @yao-msft in https://github.com/microsoft/winget-cli/pull/1731
* Make telemetry logger per context and send summary event on destruction by @yao-msft in https://github.com/microsoft/winget-cli/pull/1718
* Update Notice from latest component governance output by @yao-msft in https://github.com/microsoft/winget-cli/pull/1755
* Added settings to select preferred/required architecture. by @jedieaston in https://github.com/microsoft/winget-cli/pull/1727
* Update README.md by @denelon in https://github.com/microsoft/winget-cli/pull/1759
* Reduce localized languages per localization team guidance by @yao-msft in https://github.com/microsoft/winget-cli/pull/1761

## New Contributors
* @exxmen made their first contribution in https://github.com/microsoft/winget-cli/pull/1580
* @MagnetarRocket made their first contribution in https://github.com/microsoft/winget-cli/pull/1700

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.0.12576...v1.2.10271

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.2.10271'>Windows Package manager 1.2.10271</a>.</em>

**Comment by Tiger348** [comment]
![Winget  exe Association](https://user-images.githubusercontent.com/109897924/181166019-a4a413c2-b655-4c70-80ed-2a6b76252c9e.png)
I'm new to winget but keen to embrace...is there a work around to negate what I've attahced herein...sorry if this is the wrong forum

**Reply by denelon** [reply]
@Tiger348 

Welcome!

Please take a look at our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting). I'm guessing either the App Installer with Windows Package Manager isn't installed, or something is going on either with environment variables or the App Execution Alias. If that doesn't help, please create a new Issue so we can help get you up and running.

**Reply by Tiger348** [reply]
Thank you for your response. Sadly I still have the same issue. I'll rasie a new issue as suggested....I've attached a screen shot demonstrating that when I type winget in solation I'm prompted via WIN10 pop up asking for an program . Whre if I type winget.exe..it funcions just fine?

![winget Issue](https://user-images.githubusercontent.com/109897924/181450423-d15fb45a-ec67-4994-8948-170737ececb6.png)
